scripts/install.sh

```diff
@@ -61,22 +61,12 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 mkdir -p $WORKDIR
 
 $SUDO apt update
-$SUDO apt install -y wget gnupg2
-
-echo "Install golang v1.12"
-# TODO: v1.12 still not available in repos
-#wget https://dl.google.com/go/go1.12.6.linux-amd64.tar.gz
-#$SUDO tar -xvf go1.12.6.linux-amd64.tar.gz
-#$SUDO mv go /usr/local
-#$SUDO ln -s /usr/local/go/bin/go /usr/bin
-
-$SUDO apt-get install -y software-properties-common
-$SUDO add-apt-repository ppa:longsleep/golang-backports
-$SUDO apt-get update
-$SUDO apt install -y golang-go
-#$SUDO ln -s /usr/bin
+$SUDO apt install -y wget gnupg2 software-properties-common
 
+# golang v1.12 still not available in repos
+$SUDO add-apt-repository ppa:longsleep/golang-backports -y
```

Comment:
Mauricio,  L75 ($SUDO apt-get update) is missing here.
Shouldn't we update the repo before installing?

Reply:
It is now at line 73. (A single update after new sources are added should be enough)

Reply:
Ops... it didn't show up in the diff :-(